safehttp/response_writer.go

```diff
@@ -36,6 +40,13 @@ func (w *ResponseWriter) ServerError(code StatusCode, resp Response) Result {
 	return Result{}
 }
 
+// Header returns the collection of headers that will be set
+// on the response. Headers must be changed before Write, WriteTemplate
```

Comment:
Headers must be set before writing a response (e.g. Write, WriteTemplate).

The responses are supposed to be written once and once-only, so we drop the "for the first time".